test/powershell/Common/TestHostCS.psm1

```diff
@@ -191,7 +196,7 @@ namespace TestHost
 
         public override void WriteLine(string value)
         {
-            Streams.ConsoleOutput.Add(value);
+            Streams.ConsoleOutput.Add("::"+value+":NewLine");

```

Comment:
Idea is that we shouldn't mix "user" chars and "code" chars. If we would use "`n" and `value` is "test`n" then we will not be able to determine whether this character from the original string or generated by code (WriteLine).
We could use "test:`n" or "test:\n" but if we look at the text of tests, it is easier to read `NewLine` and `NoNewLine`, especially since this is key information.
We only need to avoid using ':' in `value`. I believe that this is not a problem for test environment.